app/models/concerns/symbolizer.rb

```diff
@@ -0,0 +1,5 @@
+module Symbolizer
```

Comment:
Other names considered: `StringToSymbolConverter` or `FieldSymbolizer`.

Reply:
would this concern be of use outside of DocumentCaptureSessionResult?  I ask since Symbolizer doesn't seem closely tied to DocumentCaptureSessonResult; and, if other methods are added  for use by other classes, the methods intended for use by other classes would be available to DocumentCaptureSessionResult and we would then maybe have to check the class of the caller.  It almost seems like adding the #selfie_status method directly in DocumentCaptureSessionResult is a very simple solution and avoids this potential complication.